### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by gunnaraasen**

### CODE REVIEW

**Comment by beckettsean** [comment]
:+1: assuming the line protocol syntax is correct. I'm waiting for the homebrew package to update so I cannot test it myself.

File: README.md [hunk 1]
```diff
@@ -70,3 +54,6 @@ curl -G http://localhost:8086/query?pretty=true \
 * Follow the [getting started guide](http://influxdb.com/docs/v0.9/introduction/getting_started.html) to find out how to install InfluxDB, start writing more data, and issue more queries - in just a few minutes.
 * See the  [HTTP API documentation to start writing a library for your favorite language](http://influxdb.com/docs/v0.9/concepts/reading_and_writing_data.html).
 
+## Upgrading
```
Comment (otoolep) [inline README.md]: Do we need to include this information here? I understood this README was about getting someone spinning up a server quickly. Why tell them about upgrading here? It is important, just not clear if this is the best place.

Comment (gunnaraasen) [inline README.md]: You're right, upgrade info probably doesn't need to go here. There's an explanation in the release announcement and in docs.

**Comment by gunnaraasen** [comment]
@otoolep upgrade section removed. Everything else look good?

File: README.md [hunk 2]
```diff
@@ -69,4 +53,3 @@ curl -G http://localhost:8086/query?pretty=true \
 * Understand the [design goals and motivations of the project](http://influxdb.com/docs/v0.9/introduction/overview.html).
 * Follow the [getting started guide](http://influxdb.com/docs/v0.9/introduction/getting_started.html) to find out how to install InfluxDB, start writing more data, and issue more queries - in just a few minutes.
 * See the  [HTTP API documentation to start writing a library for your favorite language](http://influxdb.com/docs/v0.9/concepts/reading_and_writing_data.html).
-
```
Comment (otoolep) [inline README.md]: This last link points to the older pages. I am not sure if this is on purpose.

**Comment by otoolep** [comment]
+1, looks good.

With the much shorter write blob, I do think we have room to talk more about the advantages of the system. But that can come another day.

File: README.md (hunk 2 above)
Comment (gunnaraasen) [inline README.md]: Looks like all the links point to the new pages. Maybe I'm missing something.

**Pull request merged**